tests/util_test.py

```diff
@@ -0,0 +1,220 @@
+
+import os
+import random
+import numpy as np
+from PIL import Image
+import torch
+import open_clip
+import argparse
+
+os.environ['CUDA_VISIBLE_DEVICES'] = ''
+
+def seed_all(seed = 0):
+    torch.backends.cudnn.deterministic = True
+    random.seed(seed)
+    np.random.seed(seed)
+    torch.manual_seed(seed)
+
+def inference_text(model, batches):
+    y = []
+    with torch.no_grad():
+        for x in batches:
+            x = open_clip.tokenize(x)
```

Comment:
this needs to be changed to use the new get_tokenizer method (see readme)

Reply:
done